resources/dicts/events/ceremonies/ceremony-master.json

```diff
@@ -2789,6 +2789,375 @@
+  ],
+  "warrior_75": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "responsible",
+      "sincere",
+      "loyal"
+    ],
+    "All {PRONOUN/m_c/poss} hard work has come to this. m_c stands straight and proud, adorned with {PRONOUN/m_c/poss} new name and the congratulations of {PRONOUN/m_c/poss} Clan. Tomorrow, tomorrow {PRONOUN/m_c/subject} will return to responsibility and service and r_h. Tonight is {PRONOUN/m_c/poss} night to celebrate."
+  ],
+  "warrior_76": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "ambitious",
+      "confident"
+    ],
+    "l_n calls (old_name) forward, honoring {PRONOUN/m_c/poss} r_h with the name m_c. One day, it'll be m_c's power to grant names to the warriors of c_n. This marks the first completed goal on that quest."
+  ],
+  "warrior_77": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "troublesome",
+      "rebellious",
+      "daring",
+      "adventurous"
+    ],
+    "(old_name) can't help the way {PRONOUN/m_c/poss} tail lashes, excitement bubbling over into bouncy energy as the Clan gathers. l_n mentions {PRONOUN/m_c/poss} r_h, but what (old_name) cares about is the <i>name</i>. m_c means liberty, means control over {PRONOUN/m_c/poss} own life, means the freedom to wander and hunt on {PRONOUN/m_c/poss} own, nothing holding {PRONOUN/m_c/object} back. {PRONOUN/m_c/subject/CAP} can't wait!"
+  ],
+  "warrior_78": [
+    [
+      "warrior",
+      "prepared",
+      "alive_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "competitive",
+      "flamboyant",
+      "charismatic",
+      "righteous"
+    ],
+    "Hearing l_n speak about {PRONOUN/m_c/poss} r_h in front of all the Clan, seeing (previous_mentor)'s approving nod, p1's eyes shining with pride, p2 purring loud enough to choke on the sound, it all comes together into a moment of brilliant recognition and celebration as m_c takes on {PRONOUN/m_c/poss} warrior name."
+  ],
+  "warrior_79": [
+    [
+      "warrior",
+      "prepared",
+      "alive_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "grumpy",
+      "gloomy",
+      "cold",
+      "strange"
+    ],
+    "Even at {PRONOUN/m_c/poss} own naming ceremony, m_c doesn't love being the center of attention, the tip of {PRONOUN/m_c/poss} tail twitching with unease. But {PRONOUN/m_c/subject} still {VERB/m_c/accept/accepts} (previous_mentor)'s congratulations and {PRONOUN/m_c/poss} parents proud purrs, still smiles slightly when l_n praises {PRONOUN/m_c/poss} r_h."
```

Comment:
the second sentence reads as
But she still accepts mentor's congratulations and her parents' proud purrs, **still smiles** slightly when l_n praises her r_h
the bolded part should be written, probably to "smiling slightly"